spec/text_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'text'
+require 'twilio-ruby'
+
+describe Text do
+  it { is_expected.to respond_to :time }
+
+  it { is_expected.to respond_to :send_text}
+
+  describe '#send_text' do
+    it 'should send text when order is placed' do
+     client = double :client
+     allow(client).to receive_message_chain(:messages, :create)
+     expect(Twilio::REST::Client).to receive(:new).and_return(client)
```

Comment:
Could you be even better at stubbing here? You've made a start but this doesn't totally isolate `Text` from `Twilio`.

What about:

1. Passing `Twilio::REST::Client` as a parameter in the initializer of `Text`
2. Use `double_class("Twilio::REST::Client").as_stubbed_const